### PR TITLE
fixing docs

### LINKED ISSUE #17: Fix documentation
There is some missing information and broken links.

### PR DESCRIPTION
**Pull request by ppodevlabs**
Fix #17 

Fixed some broken links and extended a bit the capsule proxy configuration to use ssl, ingress etc.

### CODE REVIEW

**Comment by maxgio92** [comment]
Hey @ppodevlabs, thank you.

LGTM

/cc @prometherion @bsctl

File: README.md [hunk 1]
```diff
@@ -41,9 +41,10 @@ We are going to mainly use the following three tools:
 
 The material here is relatively dense. We strongly encourage you to dedicate time to walk through these instructions, with a mind to learning. We do NOT provide any "one-click" deployment here. However, once you've understood the components involved it is encouraged that you build suitable, auditable GitOps deployment processes around your final infrastructure.
 
-1. [AKS Creation with AAD integration](guides/create-aks-with-add.md)
-2. [Capsule Installation](guides/capsule-installation.md)
-3. [Setup a Multitenancy Environment](guides/multitenancy-environment.md)
+1. [AKS Creation with AAD integration](guides/01-create-aks-with-add.md)
+2. [Capsule Installation](guides/02-capsule-installation.md)
+2. [Capsule Proxy Installation](guides/03-capsule-proxy-installation.md)
+3. [Setup a Multitenancy Environment](guides/04-multitenant-environment.md)
```
Comment (prometherion) [inline README.md:47]:
```suggestion
1. [AKS Creation with AAD integration](guides/01-create-aks-with-add.md)
2. [Capsule Installation](guides/02-capsule-installation.md)
3. [Capsule Proxy Installation](guides/03-capsule-proxy-installation.md)
4. [Setup a Multitenancy Environment](guides/04-multitenant-environment.md)
```

**Review by prometherion — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (prometherion) [reply README.md:47]: @ppodevlabs may I ask you to address these changes?

The PR looks very good, just a minor thing to get it merged!

**Comment by ppodevlabs** [comment]
Sorry for the delay i missed the notification

**Pull request merged**